test-data/unit/envvars.test

```diff
@@ -0,0 +1,14 @@
+# Test cases related to environment variables
+[case testEnvvar_MYPY_CONFIG_FILE_DIR]
+# cmd: mypy --config-file=subdir/mypy.ini
+# flags: --save-failures-to=/tmp/mypy
+# To run just this test:
+#    pytest -n0 -k testEnvvar_MYPY_CONFIG_FILE_DIR
+[file bogus.py]
+FOO: int = 'x'
```

Comment:
Have you looked at the Travis-CI failure? This test fails on Python 3.5.1 because that doesn't support variable declarations. You can use type comments instead.